From 9ea6661b67df128c0931a41dc6d8b435bb062769 Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Tue, 18 Jul 2017 17:56:46 +0800
Subject: [PATCH 2161/5242] MLK-16023-04 arm64: dts: imx8mq: clean up the enet
 compatible string

commit  539eb13ccf7d92a43106954469c5cb863d51274f from
https://source.codeaurora.org/external/imx/linux-imx.git

Since i.MX8MQ ENET is the same as i.MX6SX ENET version and don't
support new features added in i.MX8QM/QXP. So remove "fsl,imx8qm-fec"
compatible string.

Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 813b7a4..c566525 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -677,7 +677,7 @@
 	};
 
 	fec1: ethernet@30be0000 {
-		compatible = "fsl,imx8qm-fec", "fsl,imx6sx-fec";
+		compatible = "fsl,imx8mq-fec", "fsl,imx6sx-fec";
 		reg = <0x0 0x30be0000 0x0 0x10000>;
 		interrupts = <GIC_SPI 118 IRQ_TYPE_LEVEL_HIGH>,
 			<GIC_SPI 119 IRQ_TYPE_LEVEL_HIGH>,
-- 
1.7.9.5

